ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/ExampleClass.java

```diff
@@ -0,0 +1,6 @@
+package edu.usb.argos.ASTProcessor;
```

Comment:
What is the purpose of this class?

Reply:
The purpose was to get the methods of this example class to see the functionality in the `AstProcessorApplication` class by making prints, but I think it would be good to test it only in the tests, right?